docs/rules/wrap-iife.md

```diff
@@ -18,12 +18,18 @@ This rule requires all immediately-invoked function expressions to be wrapped in
 
 ## Options
 
-The rule takes one option which can enforce a consistent wrapping style:
+This rule has two options, a string option and an object option.
```

Comment:
I don't particularly care, but I copied this from `quotes` to be consistent with other rules: https://github.com/eslint/eslint/blame/master/docs/rules/quotes.md#L25

Which should I follow?